patch_rebaser/patch_rebaser.py

```diff
@@ -1,3 +1,164 @@
+#!/usr/bin/env python3
```

Comment:
This might be an issue if we are running this on a CentOS 7 or RHEL 7 VM.

Reply:
Going Python 3 only was intentional, as we only plan to use the script for building Stein and later. There's several constructs that are specific to Python 3 (configparser import, f-type logging, possibly others) so hopefully this will work out... Do you reckon it is possible CentOS/RHEL7 may be used for building Stein as well, going forward?